src/CistromeHGWConsumer.js

```diff
@@ -96,27 +99,31 @@ export default function CistromeHGWConsumer(props) {
      * to sibling `viewport-projection-horizontal` track IDs.
      */
     const onViewConfig = useCallback((newViewConfig) => {
-        const newTrackInfos = getHMTrackIdsFromViewConfig(newViewConfig);
-        const newSiblingTrackInfos = {};
-        for(let trackInfo of newTrackInfos) {
-            newSiblingTrackInfos[trackInfo.trackId] = getSiblingVPHTrackIdsFromViewConfig(newViewConfig, trackInfo.viewId);
+        const newTrackIds = getHMTrackIdsFromViewConfig(newViewConfig);
+        
+        // Add viewport projection horizontal track IDs for each view.
+        const newViewportTrackIds = {};
+        Array.from(new Set(newTrackIds.map(d => d.viewId))).map((viewId, i) => {    
```

Comment:
```suggestion
        Array.from(new Set(newTrackIds.map(d => d.viewId))).forEach((viewId, i) => {    
```

Are you ok if we change this to `forEach` since it has a side-effect?

Reply:
Yes, that looks nicer.